lib/chef/knife/search.rb

```diff
@@ -73,19 +73,18 @@ class Search < Knife
 
       def run
         read_cli_args
-        fuzzify_query
 
         if @type == "node"
           ui.use_presenter Knife::Core::NodePresenter
         end
 
         q = Chef::Search::Query.new
-        escaped_query = Addressable::URI.encode_component(@query, Addressable::URI::CharacterClasses::QUERY)
```

Comment:
it looks like this was fixed here first (nuo yan era fix), then fixed correctly in Chef::Search::Query but was never removed here.